src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx

```diff
@@ -10,6 +10,8 @@ import InlineSvg from 'app/components/inlineSvg';
 import Panel from 'app/components/panels/panel';
 import {getDisplayValue, getDisplayText} from './utils';
 
+const TABLE_ROW_HEIGHT = 41; // Includes 1px for border
```

Comment:
We may want to follow up with a "cell height" = 40, and "table row height" = cell height + border height, it'd be more self documenting